x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/job_messages_pane.tsx

```diff
@@ -16,17 +20,28 @@ export const JobMessagesPane: FC<JobMessagesPaneProps> = ({ jobId }) => {
   const [messages, setMessages] = useState<JobMessage[]>([]);
   const [isLoading, setIsLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
+  const {
+    services: {
+      notifications: { toasts },
+    },
+  } = useMlKibana();
 
   const fetchMessages = async () => {
     setIsLoading(true);
     try {
       setMessages(await ml.jobs.jobAuditMessages(jobId));
       setIsLoading(false);
-    } catch (e) {
+    } catch (error) {
       setIsLoading(false);
-      setErrorMessage(e);
-      // eslint-disable-next-line no-console
-      console.error('Job messages could not be loaded', e);
+      const toastNotificationService = toastNotificationServiceProvider(toasts);
```

Comment:
It'd better to use `useToastNotificationService` instead

Reply:
Thanks Dima. Updated here https://github.com/elastic/kibana/pull/79792/commits/dc1daf93666d2faff2c8407e96d03c0a854d1be6.